packages/terra-site/src/examples/card/CardContentCentered.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Card from 'terra-card';
+
+const CardContentCentered = () => (
+  <div>
+    <Card>
+      <Card.Body isContentCentered>
+         Content is centered
+      </Card.Body>
+    </Card>
+  </div>
```

Comment:
Does this need to be wrapped in a div?

Reply:
it doesn't have to be wrapped in a div I will update

Reply:
updated thanks!